hles/extraction/src/test/scala/org/broadinstitute/monster/dap/HLESExtractionPipelineBuilderIntegrationSpec.scala

```diff
@@ -69,4 +68,28 @@ class HLESExtractionPipelineBuilderIntegrationSpec extends PipelineBuilderSpec[A
   it should "successfully download records from RedCap" in {
     readMsgs(hlesOutputDir, "records/*.json") shouldNot be(empty)
   }
+
+  it should "contain records that all have a record, field_name, and value key" in {
+    readMsgs(hlesOutputDir, "records/*.json").foreach(msg => {
+      val record = msg.obj.toString()
```

Comment:
🤔 I think we can make some stronger assertions here by not deserializing to a string but rather leaving the message as an object and comparing to an expected object, much like we do in the ExtractionPipelineBuilderSpec [here](https://github.com/DataBiosphere/dog-aging-ingest/blob/e3deb8a07bab3b9febc8a6f73f030856a434bd39/hles/extraction/src/test/scala/org/broadinstitute/monster/dap/ExtractionPipelineBuilderSpec.scala#L37-L41)

Reply:
+1 to this, directly accessing keys and values avoids accidental false positives from the string we're looking for showing up in an unexpected part of the string, but not in the field we want to check

Reply:
Tests have been changed to compare to the expected object and to compare the fields and values that we are looking for